config/initializers/devise.rb

```diff
@@ -244,7 +244,16 @@
                   Rails.application.secrets[:google_client_id],
                   Rails.application.secrets[:google_client_secret],
                   {
-                    scope: "profile,email,https://www.googleapis.com/auth/youtube.readonly"
+                      name: 'register_youtube_channel',
```

Comment:
we should update the corresponding part in the README where we describe the oauth redirect URLs